### DISCUSSION TITLE
Camera & Microphone not detected / not working & API Signature failed error on SEB 3.2.5 on Mac OS Catalina & Monterey

### DISCUSSION THREAD

**Discussion by Chris251290** [SafeExamBrowser/seb-mac#304] [Q&A]
Hi Team,

"Scenario 1":
We have installed **SEB version 3.2.5** on **Mac OS Catalina** Laptop. While opening the SEB Launch file to attend the proctoring exam, Camera and Microphone not working or not detected on Safe exam browser.
But, Camera and Microphone working normally on other applications like Zoom, Jitsi or Google Chome.

We have allowed Safe Exam Browser in **Mac laptop Settings - > Security & Privacy -> Camera, Microphone & Accessibility options**. But, still not working on Safe exam browser. 
There are no Antivirus installed on the laptop.

Do I need to allow the Camera & Microphone in any other settings in Mac Laptop.

"Scenario 2":
Another laptop MAC version Catalina & SEB Version 3.2.5. While starting the exam using Start Exam button, It shows the "API Signature failed" error.

("Note" : Once we updated the MAC OS to Ventura 13.1 or 13.4, Safari browser version also updated to 16.2 & 16.5 respectively. Then the above 2 scenarios are fixed and the SEB is working fine on latest version of MAC OS.)

Kindly assist me and let us know why these issues are coming in the previous "MAC OS versions Catalina & Monterey" .

Thanks in advance.

**Comment by Chris251290** [comment]
Hi @danschlet   Can you please look into this and reply back for our queries?

**Reply by danschlet** [reply]
Please read the download page and release notes. Apple supports camera/microphone access in WebViews only starting macOS Big Sur. 
macOS 11.15 Catalina is no longer supported with security updates by Apple, so not recommend anyways.
